source/includes/driver-compatibility-caveat.rst

```diff
@@ -0,0 +1,6 @@
+.. note::
+
+   The compatibility matrices show which versions of the driver are
+   recommended for use with a given major release. Older driver versions
+   may be able to connect to a version of MongoDB which is not officially
+   supported, but not all features will be available.
```

Comment:
Yes, that's correct. It says older versions *may* be able to connect to unsupported versions, which is true and was brought up in DOCS-9400 as well as in my conversations with Shane Harvey about what exactly we mean by compatible and incompatible. He pointed out that old versions of pymongo can still connect to MongoDB version 3.4 and perform some kinds of operations, and that it's frequently possible to upgrade a MongoDB server without upgrading the driver (or changing application code) without any ill effects. He mentioned that they test PyMongo 2.9 against MongoDB 2.4, 2.6, 3.0, 3.2, 3.4, and 3.5, even though it's only officially supported for 3.0 and lower.

All that being said, if you think this paragraph is going to confuse more people than it helps, I'm happy to spike it altogether and leave this with just the minimum viable ticket, which is correcting the python.txt page so that its compatibility matrix agrees with the main driver compatibility page.